.github/workflows/lint.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Maven compile
```

Comment:
I think rebasing on `master` might eliminate a lot these files with the small changes that Katie already made.